charts/ziti-controller/values.yaml

```diff
@@ -396,3 +396,29 @@ ingress-nginx:
     extraArgs:
       # -- configure subchart ingress-nginx to enable the pass-through TLS feature
       enable-ssl-passthrough: "true"
+
+# -- allow the insertion of raw/unmodified config blocks
```

Comment:
Let's name it like `finalConfig` or `configFooter` or `tailYaml` or `endRawConfig` or `trailingConfig` (or insert better name) and highlight with unavoidable clarity in the doc comment that the raw config are tacked on to the end/bottom of the config YAML and must not collide with any existing config. 

This doesn't override anything, and any top-level config properties already defined in the ConfigMap template can't be used in this trailing config. If an existing, templated key is re-used in this section then ziti will panic abort.